### ISSUE TITLE
Hornet should pad the tag

### ISSUE THREAD

**Issue by GalRogozinski** [iotaledger/hornet#659]
**Describe the bug**
Hornet should expand `tag` trytes parameter to 27 trytes, or return an error if the `tag` is too long.

**To Reproduce**
Issue the following command via api:
```
{
 "command": "findTransactions",
 "tags": ["CCYZC"]
}
```
The following is returned:
```
{
    "error": "Tag invalid length: CCYZC"
}
```

**Expected behavior**
Hornet should pad the tag with 9s to get a length of 27 trytes

**Additional context**
This bug may break some applications that used the tangle before Chrysallis

**Comment by muXxer** [comment]
The user should know that he is doing something wrong... didn't we had this discussion already?

**Comment by GalRogozinski** [comment]
yeah and the conclusion was to pad if I remember correctly

you said someone should open an issue an no one did
so now I opened :-)

**Comment by muXxer** [comment]
I really hate that decision.... but whatever, I will "fix" it.
The old API is so broken by design.... what a mess :D

**Comment by muXxer** [comment]
"Fixed" in #657
